### PR TITLE
HMI occupancy table fixes

### PR DESCRIPTION
**Pull request by jacksongonzales**
per Yindi's comments on [DAH-1173](https://sfgovdt.jira.com/jira/software/c/projects/DAH/boards/122?modal=detail&selectedIssue=DAH-1173)

### CODE REVIEW

File: app/assets/json/translations/react/en.json [hunk 1]
```diff
@@ -821,9 +821,8 @@
   "listings.includesPriorityUnits": "Includes Priority Units for %{priorities}",
   "listings.includingChildren": "(including <span class='eligibility-subtext'>%{number} %{children}</span> under 6)",
   "listings.incomeExceptions.intro": "People in your household may need <a href='%{url}' target='_blank'>special income calculations</a> if they:",
-  "listings.incomeExceptions.nontaxable": "Receive <span data-tooltip='%{tooltip}' title='%{tooltip}' aria-haspopup='true' class='has-tip'>non-taxable income</span>",
-  "listings.incomeExceptions.nontaxableTooltip": "Non-taxable income might include SSI, SSDI, child support payments, and worker’s compensation benefits.",
-  "listings.incomeExceptions.students": "Are full-time students (but not the primary applicant)",
+  "listings.incomeExceptions.nontaxable": "Receive non-taxable income (non-taxable income might include SSI, SSDI, child support payments, and worker’s compensation benefits).",
```
Comment (kramduckner) [inline app/assets/json/translations/react/en.json:824]: should these changes be applied to other languages?

**Review by kramduckner — COMMENTED** [review]
looking so good! Just that translation comment and this one question - is it ok is the special income calculations link doesn't work in the deployed version?
Screen Shot 2022-08-04 at 7 44 33 PM

File: app/assets/json/translations/react/en.json (hunk 1 above)
Comment (jacksongonzales) [reply app/assets/json/translations/react/en.json:824]: oop, that link above should prob not be broken 😅 good catch. for those string changes, they are dropped into the document we have for keeping track of new/updated translations we're gonna order. I assume we'll do a lump PR for filling in those translations once we get them back, so I think better to wait for those than try my bad guess at how they should translate 😄

Comment (kramduckner) [reply app/assets/json/translations/react/en.json:824]: re: strings Makes sense! Thanks for fixing that link. I'll approve.

**Review by kramduckner — APPROVED** [review]

**Pull request merged**